test/integration/test_run_reports.py

```diff
@@ -0,0 +1,22 @@
+from src.config import Config, ReportConfig
+from src.orderlyweb_client_wrapper import OrderlyWebClientWrapper
+from src.utils.run_reports import run_reports
+
+
+def test_run_reports_handles_error():
```

Comment:
This test was previously in `test_task_run_diagnostic_reports` (https://github.com/vimc/montagu-task-queue/pull/12/files#diff-df5f97f7f8fb818f069a53a52dc3aab3L76) but is actually a test of the generic `run_reports` method, so moved to it's own file.